components/containers/unordered/tests/unit/CMakeLists.txt

```diff
@@ -10,8 +10,13 @@ set(tests
 
 set(unordered_map_FLAGS
     COMPONENT_DEPENDENCIES unordered)
+
+if(HPX_WITH_NETWORKING)
+  set(__localities LOCALITIES 2)
+endif()
```

Comment:
Wouldn't it be better to perform this check here
https://github.com/STEllAR-GROUP/hpx/blob/master/cmake/HPX_AddTest.cmake#L15

maybe even not add the test at all if networking was disabled?

Reply:
> Wouldn't it be better to perform this check here
https://github.com/STEllAR-GROUP/hpx/blob/master/cmake/HPX_AddTest.cmake#L15

Good thinking!

> maybe even not add the test at all if networking was disabled?

I think that all of these tests run perfectly well on one locality, just with reduced test coverage. We might have to remove some of them, but I would like to be selective about this.

Reply:
> Wouldn't it be better to perform this check here
https://github.com/STEllAR-GROUP/hpx/blob/master/cmake/HPX_AddTest.cmake#L15

Actually, this is already in place: https://github.com/STEllAR-GROUP/hpx/blob/master/cmake/HPX_AddTest.cmake#L55-L59